devtools/common/src/main/java/io/quarkus/cli/commands/CreateProject.java

```diff
@@ -92,6 +99,10 @@ public boolean doCreateProject(final Map<String, Object> context) throws IOExcep
         context.put(SOURCE_TYPE, sourceType);
         context.put(BUILD_TOOL, buildTool);
 
+        if (extensions != null && extensions.stream().anyMatch(e -> e.toLowerCase().contains("spring-web"))) {
```

Comment:
Could you please confirm that I keep this change following the [@ia3andy remark](https://github.com/quarkusio/quarkus/pull/3492#discussion_r313438158)

Reply:
Hi @gsmet,

could you pelase confirm what should I do with this? Not sure I understand

Reply:
@aureamunoz I think what @gsmet means is that we should keep it as it is for now, but also create an issue for further improvements in the future.
The idea would be to pass a list of the active extensions to `CreateProject` so it can then decide what needs to be done without having to do this sort if check (which would be pretty much the same for each new extension that needs some custom functionality).
Having said that, if you have time to do it as part of this PR and it's not too much of a change, then feel free to do it :)

Reply:
@aureamunoz can you open an issue to explore code improvements based on this suggestion?

Reply:
done. New issue [here](https://github.com/quarkusio/quarkus/issues/3674)